cli/daemon/run/run.go

```diff
@@ -349,7 +349,7 @@ func (r *Run) startProc(params *startProcParams) (p *Proc, err error) {
 		Meta:      params.Meta,
 		exit:      make(chan struct{}),
 		buildDir:  params.BuildDir,
-		log:       r.log.With().Str("procID", pid).Logger(),
+		log:       r.log.With().Str("procID", pid).Str("buildDir", params.BuildDir).Logger(),
```

Comment:
This is just to make it easier to find the generated code while debugging runtime issues